gnucash/register/register-gnome/combocell-gnome.c

```diff
@@ -664,9 +671,11 @@ gnc_combo_cell_modify_verify (BasicCell* _cell,
             unblock_list_signals (cell);
         }
         gnc_basic_cell_set_value_internal (_cell, newval);
-        block_list_signals (cell);
+
+        block_list_signals (cell); //Prevent recursion
         gnc_item_list_select (box->item_list, NULL);
         unblock_list_signals (cell);
+
```

Comment:
How about
```
    block_list_signals (cell); //Prevent recursion       
    if (cell-&gt;shared_store &amp;&amp; gnc_item_list_using_temp (box-&gt;item_list))
    {
          gnc_item_list_set_temp_store (box-&gt;item_list, NULL);
          gtk_list_store_clear (box-&gt;tmp_store);
     }
     gnc_item_list_select (box-&gt;item_list, NULL);
     unblock_list_signals (cell);
    gnc_basic_cell_set_value_internal (_cell, newval);
```
Or does `gnc_basic_cell_set_value_internal (_cell, newval);` need to happen in between clearing the list store and clearing the selection?

Reply:
Did not see that, that part is dealing with when no type-ahead/quickfill found.
Made the change locally and after testing could not see any problems so will add it to PR and push this tomorrow.